src/ducks/settings/TagsList.jsx

```diff
@@ -0,0 +1,52 @@
+import React, { Fragment } from 'react'
+import PropTypes from 'prop-types'
+import Fuse from 'fuse.js'
+import orderBy from 'lodash/orderBy'
+import useBreakpoints from 'cozy-ui/transpiled/react/hooks/useBreakpoints'
+
+import Divider from 'cozy-ui/transpiled/react/MuiCozyTheme/Divider'
+import List from 'cozy-ui/transpiled/react/MuiCozyTheme/List'
+import TagListItem from 'ducks/settings/TagListItem'
+
+const fuse = new Fuse([], {
+  findAllMatches: true,
+  threshold: 0.3,
+  keys: ['label']
+})
+
+const listStyle = { padding: '.5rem 0' }
+
+const TagsList = ({ tags, filter, sort }) => {
+  const { isMobile } = useBreakpoints()
+  const [sortKey, sortOrder] = sort.split('-')
+  let filteredTags = tags
+  if (filter.length !== 0) {
+    fuse.setCollection(filteredTags)
+    filteredTags = fuse.search(filter).map(result => result.item)
+  }
+  const sortedTags = orderBy(filteredTags, [sortKey], [sortOrder])
+  return (
+    <>
+      {isMobile && sortedTags.length !== 0 && <Divider />}
+      <List style={listStyle}>
+        {sortedTags.map((tag, index) => {
+          return (
+            <Fragment key={`${tag._id} ${index}`}>
```

Comment:
why do we need to use index? _id can be the same? can't we use another prop in tag (in case _id are not unique)?